app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -36,10 +36,11 @@ def redirect_if_blank_phone
     end
 
     def confirm_multiple_factors_enabled
-      return if UserSessionContext.confirmation_context?(context) || phone_enabled?
+      phone_enabled = phone_enabled?
+      return if UserSessionContext.confirmation_context?(context) || phone_enabled
```

Comment:
I dunno if short-circuiting would have saved us anything previously, but by assigning this variable, we're no longer bailing based on the result of `confirmation_context?`, which I assume would be a pretty trivial check.

If we cared for short-circuiting, I would have expected something like...

```suggestion
      return if UserSessionContext.confirmation_context?(context)
      phone_enabled = phone_enabled?
      return if phone_enabled
```

Reply:
ahhhh, shoot, thank you

Reply:
https://github.com/18F/identity-idp/pull/7002